src/components/VoteCounter.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+function VoteCounter({ article_id, votes, voteUpdater }) {
+    const [currentVotes, setCurrentVotes] = useState(votes);
+    const [responseMsg, setResponseMsg] = useState(null);
+    const [disableButtons, setDisableButtons] = useState(false);
+
+    function updateVote(voteChange) {
+        setDisableButtons(true);
+        setResponseMsg("Waiting for confirmation");
```

Comment:
It's good to let the use know that the vote is loading, but they load in so quickly that it's almost not worth showing a whole message for it, as it looks kind of glitchy having that message appear and quickly disappear - up to you if you prefer this behavior or not!

Imo, disabling the button while the request loads is good enough feedback